jll/N/Notcurses_jll/Compat.toml

```
@@ -4,6 +4,9 @@ julia = "1.6.0-1"
 ["3-3.0.11"]
 JLLWrappers = "1.2.0-1"
 
+["3.0.9-3"]
+FFMPEG_jll = "6"
```

```suggestion
FFMPEG_jll = "6.1.2"
```
to match https://github.com/JuliaPackaging/Yggdrasil/pull/10314/files#diff-e096efc0a09963158c9b75da03b10636c59ca30447fa6e5b295a0f7a17ac6bb1R88